cmd/client-fs.go

```diff
@@ -671,8 +671,12 @@ func (f *fsClient) listInRoutine(contentCh chan<- *clientContent) {
 					continue
 				}
 				if os.IsNotExist(e) {
+					// Lstat makes no attempt to follow the broken link.
+					_, e = os.Lstat(filepath.Join(fpath, fi.Name()))
```

Comment:
fi is null here when there is an error due to L666.

Reply:
Done :+1: